pages/pipelines/notifications.md.erb

```diff
@@ -23,7 +19,39 @@ notify:
 ```
 {: codeblock-file="pipeline.yml"}
 
-You can trigger notifications as emails, as a message in a Basecamp Campfire, or as a message in a Slack Channel.
+Available notification types:
+<table>
+  <tr>
+    <td><code>Email</code></td>
+    <td>
+      Send an email to the specified email address.<br>
+      <em>Required attributes:</em> <code>email</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>Basecamp</code></td>
+    <td>
+      Post a message to a Basecamp Campfire. Requires a Basecamp Chatbot to be configured in your Basecamp organization.<br>
+      <em>Required attributes:</em> <code>basecamp_campfire</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>Slack</code></td>
+    <td>
+      Post a message to the specified Slack Channel. Requires a Slack notification service to be enabled for each channel.<br>
+      <em>Required attributes:</em> <code>slack</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>Webhooks</code></td>
+    <td>
+      Send a notification to the specified webhook URL.<br>
+      <em>Required attributes:</em> <code>webhook</code>
+    </td>
+  </tr>
+</table>
```

Comment:
This currently looks like this:

<img width="296" alt="image" src="https://user-images.githubusercontent.com/282113/87084451-1c044400-c1e3-11ea-8944-007f5a990536.png">

I wonder if we should just make the names lowercase for now? Also maybe we make the required attributes a list, with a little description?

Reply:
Lowercase would look way better here 👍🏻 I think I'd rather leave the required attributes section as-is, otherwise the doc gets pretty repetitive. I think this sidebar is really just a quick reference, if people want more info about stuff I'd rather they clicked through to the actual page 😄